### ISSUE TITLE
csv data set

### ISSUE THREAD

**Issue by Arpit25upadhyay** [abdulfatir/twitter-sentiment-analysis#24]
how to insert dataset in preprocessing,py file

**Comment by poojadabake11** [comment]
df=pd.read_csv("csv file path")

**Comment by Arpit25upadhyay** [comment]
![Screenshot (18)](https://user-images.githubusercontent.com/54313404/63795927-abce3600-c922-11e9-9ced-d53ecefdfabe.png)
this is my code now where and how to add csv file

**Comment by sheth17nirali** [comment]
> df=pd.read_csv("csv file path")

hello. can you pls help me on how to import both test and train dataset using panda and where we need to make changes to the code.
thankyou

**Comment by sauravjoshi23** [comment]
Hey  @sheth17nirali  , once you have preprocessed the entire dataset by running preprocess.py, the next step is to split the data into training and testing. This can be done by : 

**from sklearn.model_selection import train_test_split
train, test = train_test_split(processed_data, test_size=0.2, random_state=42, shuffle=True)

train.to_csv('train-processed.csv', sep=',', index=False)
test.to_csv('test-processed.csv', sep=',', index=False)**

Saving them to their respective files so that they can be used later for modeling purposes.

**Comment by DriSwee** [comment]
Hello, may I ask where I can get the csv file?

**Comment by abdulfatir** [comment]
As mentioned in the readme, the dataset is not publicly available.
